### PR TITLE
Rename committed RangeIterator to UnmarshalIterator

### PR DESCRIPTION
**Pull request by nopcoder**

### CODE REVIEW

File: graveler/committed/meta_range_manager.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ type MetaRangeManager interface {
 	// over the MetaRange from the first value >= from
 	NewMetaRangeIterator(ns graveler.StorageNamespace, metaRangeID graveler.MetaRangeID, from graveler.Key) (Iterator, error)
 
-	// NewRangeIterator accepts a Range ID, and returns a ValueIterator
+	// NewUnmarshalIterator accepts a Range ID, and returns a ValueIterator
```
Comment (johnnyaug) [inline graveler/committed/meta_range_manager.go:46]: revert

File: graveler/committed/range_manager.go [hunk 2]
```diff
@@ -29,7 +29,7 @@ type RangeManager interface {
 	// If path not found, (nil, ErrPathNotFound) is returned.
 	GetValue(ns Namespace, key Key, pid ID) (*Record, error)
 
-	// NewRangeIterator takes a Range ID and returns an ValueIterator seeked to >= "from" value
+	// NewUnmarshalIterator takes a Range ID and returns an ValueIterator seeked to >= "from" value
```
Comment (johnnyaug) [inline graveler/committed/range_manager.go:32]: revert

**Review by johnnyaug — CHANGES_REQUESTED** [review]

**Review by johnnyaug — APPROVED** [review]
:pray:

**Comment by nopcoder** [comment]
> 🙏

Thanks!

**Pull request merged**